src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/PanelSummary/Popup.tsx

```diff
@@ -9,7 +9,7 @@ interface PopupProps {
 export default function Popup({ trigger, children }: PopupProps) {
   return (
     <ReactPopup
-      trigger={trigger}
+      trigger={<>{trigger}</>}
```

Comment:
This might look weird, but it's to deal with the `function components cannot be given refs` error. I'm not 100% clear on why this solves it, but it does. Perhaps somebody else can better explain this?